patches/api/0323-Stinger-API.patch

```diff
@@ -0,0 +1,47 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Tue, 22 Jun 2021 23:16:11 -0400
+Subject: [PATCH] Stinger API
+
+
+diff --git a/src/main/java/org/bukkit/entity/LivingEntity.java b/src/main/java/org/bukkit/entity/LivingEntity.java
+index cda05df6784dd4d6a09710a416dcb71c016dabfc..06812f127e29b5c4cd4328a09b822cdb880ca4b3 100644
+--- a/src/main/java/org/bukkit/entity/LivingEntity.java
++++ b/src/main/java/org/bukkit/entity/LivingEntity.java
+@@ -336,6 +336,36 @@ public interface LivingEntity extends Attributable, Damageable, ProjectileSource
+      */
+     public void setArrowsInBody(int count);
+ 
++    // Paper Start - Stinger API
++    /**
++     * Gets the time in ticks until the next stinger leaves the entity's body.
```

Comment:
What is a stinger? Add to the docs.

Reply:
![image](https://user-images.githubusercontent.com/23108066/125980503-3b16f503-4be7-481c-8e19-ce61b9aad9b0.png)

Stingers appear in your body when you get stung by bees, similar to arrows. Should I just document it as bee stinger? Cause I am not sure.

Reply:
Yeah, as simple as just adding `bee` in front of every `stinger` in this PR would definitely be enough; that should at least be more Googleable if it still doesn't click.